geopy/distance.py

```diff
@@ -35,40 +51,47 @@
                   'GRS-67':        (6378.1600,   6356.774719,   1 / 298.25),
                   }
 
-Here's an example usage of distance.vincenty::
+Here are examples of distance.distance::
 
-    >>> from geopy.distance import vincenty
+    >>> from geopy import distance
     >>> newport_ri = (41.49008, -71.312796)
     >>> cleveland_oh = (41.499498, -81.695391)
-    >>> print(vincenty(newport_ri, cleveland_oh).miles)
-    538.3904451566326
+    >>> print(distance.distance(newport_ri, cleveland_oh).miles)
+    538.390445368
+
+    >>> wellington = (-41.32, 174.81)
+    >>> salamanca = (40.96, -5.50)
+    >>> print(distance.distance(wellington, salamanca).km)
+    19959.6792674
+
+The second example above fails unless geographiclib has been installed.
```

Comment:
By `fails` you mean that there's a significant error, right? Maybe you could show a difference by also calling `print(distance.vincenty(wellington, salamanca).km)` there?